### DISCUSSION TITLE
Can I get balance sample in case level?

### DISCUSSION THREAD

**Discussion by Bigsealion** [TorchIO-project/torchio#768] [Q&A] [answered]
Hi:
    In my datasets, most of the parts have label 1, and small parts have labels 1 and 2, which creates a huge imbalance in training.
    So, I want to get a balanced sample (subject with label 1 and subject with labels 1&2 have the same number) in each epoch. I have known `Torchio` can get a balanced sample in patch level by `LabelSampler`, but can `Torchio` create a balanced sample in case level easily?

Thanks!

**Comment by fepegar** [comment, marked as answer]
This is not implemented at the moment, but I think you can use a PyTorch [`Sampler`](https://pytorch.org/docs/stable/data.html#torch.utils.data.Sampler) for this.

**Reply by Bigsealion** [reply]
Thanks for your reply! It's really helpful!
